examples/src/main/java/alluxio/cli/TestRunner.java

```diff
@@ -45,6 +46,10 @@
   @Parameter(names = {"-h", "--help"}, description = "Prints usage information", help = true)
   private boolean mHelp;
 
+  @Parameter(names = "--directory",
+      description = "The directory to test (can be a non-root mount point). Default: /" )
```

Comment:
If you assign `mDirectory = false`, it should print the default value automatically